plugins/ca-kafka-local/plugin.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "ca-kafka-local",
+  "version": "0.0.1",
+  "description": "Environment variables and tools to interact with `kafka-local` in your service",
+  "readme": "README.md",
+  "env": {
+    "KAFKA_BROKERS": "localhost:14231",
```

Comment:
is this always going to be the ports for Kafka? as there is some standardized Kafka ports, could we not use them?

Reply:
@cultureamp-brucew mentioned on Slack, but we have set unique ports for everything and registered them in https://github.com/cultureamp/local-ops.

We've tried to avoid default or common ports so that our Culture Amp stuff can start without conflicting with something else you're running on your system.